src/pages/billing/List.js

```diff
@@ -116,12 +115,9 @@ const BillingProjectActions = ({ project: { projectName }, loadProjects }) => {
 }
 
 const ProjectListItem = ({ project, project: { projectName, roles, status, message, cloudPlatform }, loadProjects, isActive }) => {
-  const cloudContextIcon =
-    div({ style: { display: 'flex', marginRight: '0.5rem' } }, [
-      Utils.switchCase(cloudPlatform,
-        [cloudProviders.gcp.label, () => h(CloudGcpLogo, { title: cloudProviders.gcp.iconTitle, role: 'img' })],
-        [cloudProviders.azure.label, () => h(CloudAzureLogo, { title: cloudProviders.azure.iconTitle, role: 'img' })])
-    ])
+  const cloudContextIcon = !!cloudPlatform && div({ style: { display: 'flex', marginRight: '0.5rem' } }, [
+    h(CloudProviderIcon, { cloudProvider: cloudPlatform })
```

Comment:
`cloudPlatform` will be 'UNKNOWN', not falsey, if we can't determine the platform.

Reply:
@nawatts pushed up a fix.

Reply:
Thanks! Does this also need to be handled in the new workspace modal?

Reply:
I don't think so because we don't show billing projects in an error status in the new workspace modal. But we could add the check there to be safe… I don't feel strongly either way.

Reply:
Great. I'll vote against adding complexity if we don't need it.

Reply:
Also pushed up test coverage (verified modified tests bomb without the `isCloudProvider` check, and passed locally for me with the check).